WebhookNotificationActivity/index.ts

```diff
@@ -44,14 +53,26 @@ const notificationModel = new NotificationModel(
 // Whether we're in a production environment
 const isProduction = process.env.NODE_ENV === "production";
 
+// Webhook must be an https endpoint
+const abortableFetch = AbortableFetch(agent.getHttpsFetch(process.env));
+// 10 seconds timeout by default
+const fetchWithTimeout = setFetchTimeout(
+  (process.env.FETCH_KEEPALIVE_TIMEOUT
```

Comment:
I'm not sure this makes sense, `FETCH_KEEPALIVE_TIMEOUT` sets the [timeout on the socket](https://github.com/node-modules/agentkeepalive#new-agentoptions) while `setFetchTimeout` sets the timeout on the HTTP(s) request - I believe the former should be much higher than the latter (in fact it defaults to 30s)

Reply:
what do you suggest as request timeout value ?

Reply:
I would look at the 95th percentile response time of the webhook endpoint and add 20% to that